src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TeamOperations/CreateEdit.js

```diff
@@ -254,6 +261,17 @@
     return result;
   };
 
+  function checkIfPlayerExists(playersWhichWasInTeam, playerId) {
+    var ifexist = false;
+    for (var i = 0; i < playersWhichWasInTeam.length; i++) {
+      if (+playersWhichWasInTeam[i].id === playerId) {
```

Comment:
what do you need this + at the beggining of the condition for?

Reply:
https://medium.com/@nikjohn/cast-to-number-in-javascript-using-the-unary-operator-f4ca67c792ce